pkg/asset/tls/adminkubeconfig.go

```diff
@@ -73,39 +73,6 @@ func (c *AdminKubeConfigSignerCertKey) Name() string {
 	return "Certificate (admin-kubeconfig-signer)"
 }
 
-// AdminKubeConfigCABundle is the asset the generates the admin-kubeconfig-ca-bundle,
-// which contains all the individual client CAs.
-type AdminKubeConfigCABundle struct {
-	CertBundle
-}
-
-var _ asset.Asset = (*AdminKubeConfigCABundle)(nil)
-
-// Dependencies returns the dependency of the cert bundle.
-func (a *AdminKubeConfigCABundle) Dependencies() []asset.Asset {
-	return []asset.Asset{
-		&AdminKubeConfigSignerCertKey{},
-		&KubeAPIServerLBSignerCertKey{},
```

Comment:
nit:
KAS can use this file to trust the client certificate generated by admin-kubeconfig-signer. just drop kubeapiserver*signer

Reply:
I think I deleted these lines.  We're trying to create a call bundle to verify the KAS serving cert, not one to verify clients.

Reply:
Oh, I understand now.  Sure, I'll make it client ca only.  Lgty otherwise?  An lgtm and hold so I can get it this weekend?